txn.go

```diff
@@ -28,9 +29,21 @@ import (
 var logger = loggo.GetLogger("juju.txn")
 
 const (
-	// nrRetries is the number of time a transaction will be retried
-	// when there is an invariant assertion failure.
-	nrRetries = 3
+	// defaultClientTxnRetries is the default number of times a transaction will be retried
+	// when there is an invariant assertion failure (for client side transactions).
+	defaultClientTxnRetries = 3
+
+	// defaultServerTxnRetries is the default number of times a transaction will be retried
+	// when there is an invariant assertion failure (for server side transactions).
+	defaultServerTxnRetries = 50
+
+	// defaultRetryBackoffMillis is the default interval used to pause between
+	// unsuccessful transaction operations.
+	defaultRetryBackoffMillis = 1
+
+	// defaultRetryFuzzMillis is the default fudge factor used when pausing between
+	// unsuccessful transaction operations.
+	defaultRetryFuzzMillis = 1
```

Comment:
I believe for Fuzz we usually use a percentage (like 20%) vs a specific count.
I also wonder whether these should be 1*time.Millisecond rather than putting Millis into the name